test/simulation/install.sh

```diff
@@ -8,28 +8,38 @@ delete_dir() {
 	return 0
 }
 
-
 temp_gopath=$PWD/temp/go
 temp_lndpath=${temp_gopath}/src/github.com/lightningnetwork/lnd
-
+LND_COMMIT_HASH="aeda418ea81bdb5c80a07b606e8da2de5b85852e"
 if [ -f ${temp_lndpath}/lnd-debug ]
+then
+  LND_VERSION=$(${temp_lndpath}/lnd-debug --version)
+else
+  LND_VERSION=""
+fi
+$(echo $LND_VERSION | grep -q "$LND_COMMIT_HASH")
+if [ $? == 0 ]
 then
     echo "lnd already installed"
 else
+    echo "deleting temporary gopath directory"
     delete_dir ${temp_gopath}
 
+    echo "getting dep..."
     GO111MODULE=off GOPATH=${temp_gopath} go get -u github.com/golang/dep/cmd/dep
```

Comment:
Do we need `dep` with the new lnd version?

Reply:
I confirmed that we do not, I'll remove. Thanks.